src/Arcus.Templates.Tests.Integration/Security/v1/SecurityHeaderTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.Templates.Tests.Integration.Fixture;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Templates.Tests.Integration.Security.v1
+{
+    [Collection("Integration")]
+    public class SecurityHeaderTests
+    {
+        private readonly TestConfig _configuration;
+        private readonly ITestOutputHelper _outputWriter;
+
+        public SecurityHeaderTests(ITestOutputHelper outputWriter)
+        {
+            _configuration = TestConfig.Create();
+            _outputWriter = outputWriter;
+        }
+
+        [Fact]
+        public async Task SecurityHeaders_BasicHttpCall_ServerHeaderNotReturned()
```

Comment:
Cool that you use the integration setup! :)

Reply:
I did a copy paste so I didn't think!

No just kidding, it's very easy to use, thanks!